<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_form</class>
 <widget class="QWidget" name="login_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>100</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>149</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_edit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>120</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>190</y>
     <width>250</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
